packages/fluentui/react-northstar/src/utils/positioner/Popper.tsx

```diff
@@ -10,6 +10,29 @@ import { getScrollParent } from './getScrollParent';
 import { getPlacement, applyRtlToOffset } from './positioningHelper';
 import { PopperModifiers, PopperProps } from './types';
 
+// https://github.com/facebook/react/blob/848bb2426e44606e0a55dfe44c7b3ece33772485/packages/react-dom/src/client/ReactDOMHostConfig.js#L157-L166
+const isAutofocusEnabled = (node: Node, reactInstanceKey: string): boolean => {
```

Comment:
```suggestion
const hasAutofocusProp = (node: Node): boolean => {
```

Reply:
BTW, it can be also `undefined`

```suggestion
const hasAutofocusProp = (node: Node): boolean | undefined => {
```

Reply:
If we will cache `getReactInstanceKey()`, we can call it inside `hasAutofocusProp` and we will not need an additional param 👍